content/en/docs/setup/install/multicluster/single-network/remote/index.md

```diff
@@ -0,0 +1,159 @@
+before you continue.
+
+## Configure Trust
+
+{{< boilerplate trust-config >}}
+
+## Deploy Istio
+
+Next, deploy Istio in `Cluster_2` with the discovery address pointing at the
+ingress gateway of `Cluster_1`. Your [initial configuration](/docs/setup/install/multicluster/single-network/initial-configuration)
+enabled mesh expansion on `Cluster_1`, and now `Cluster_2` can access the
+discovery server in `Cluster_1` via the ingress gateway.
+
+The new remote cluster requires the following configurations:
+
+<table>
+    <thead>
+    <tr>
+        <th>Configuration field</th>
+        <th>Description</th>
+        <th>Value</th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>clusterName</code>
+            </td>
+            <td>Specifies a human-readable cluster name.</td>
+            <td><code>${CLUSTER_2}</code></td>
+        </tr>
+        <tr>
+            <td><code>network</code></td>
+            <td>Specifies a network ID as an arbitrary string. All clusters in your mesh
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshID</code></td>
+            <td>Specifies a mesh ID as an arbitrary string. All clusters in your mesh share
+            the same mesh ID.</td>
+            <td><code>${MESH}</code></td>
+        </tr>
+        <tr>
+            <td><code>remotePilotAddress</code></td>
+            <td>IP address of the Istio ingress gateway of `Cluster_1`. </td>
+            <td>${DISCOVERY_ADDRESS}</td>
+        </tr>
+    </tbody>
+</table>
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy Istio in `Cluster_2` with the following steps:
+
+1. Set the value of the `DISCOVERY_ADDRESS` environment variable to the IP
+   address of the Istio ingress gateway of `Cluster_1` with the following
+   command:
+
+{{< text bash >}}
+$ export DISCOVERY_ADDRESS=$(kubectl \
+  --context=${CTX_1} \
+  -n istio-system get svc istio-ingressgateway \
+  -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+{{< /text >}}
+
+1. To pass configuration values to the Istio operator for installation, you
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+{{< text bash >}}
+$ cat <<EOF> ${WORK_DIR}/${CLUSTER_2}/install.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: ${MESH}
+      multiCluster:
+        clusterName: ${CLUSTER_2}
+      network: ${NETWORK_1}
+      # Access the control plane discovery server via the ingress
+      remotePilotAddress: ${DISCOVERY_ADDRESS}
+EOF
+{{< /text >}}
+
+1. Install Istio on `Cluster_2` with the following command:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_2} manifest apply -f \
```

Comment:
we're using "install" instead of "manifest apply" now are we not?

Reply:
I believe so.  @ostromart can you confirm?